dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/MatchingAllelesFilterProcessorTest.java

```diff
@@ -27,62 +28,56 @@
     public void setUp() throws Exception {
         filter = new MatchingAllelesFilterProcessor();
         matchingAllelesVariants = new ArrayList<>();
-        matchingAllelesVariants.add(new SubSnpCoreFields(26201546L, 1, 13677177L, 1,
-                                                         "NT_455866.1", 1766472L, 1766472L, 1, LocusType.SNP,
-                                                         "4", 91223961L, 91223961L,
-                                                         "T", "T", "A", "T/A",
-                                                         "NC_006091.4:g.91223961T>A", 91223961L, 91223961L, 1,
-                                                         "NT_455866.1:g.1766472T>A", 1766472L, 1766472L, 1));
-        matchingAllelesVariants.add(new SubSnpCoreFields(26954817L, -1, 13677177L, 1,
-                                                         "NT_455866.1", 1766472L, 1766472L, 1, LocusType.SNP,
-                                                         "4", 91223961L, 91223961L,
-                                                         "T", "T", "C", "G/A",
-                                                         "NC_006091.4:g.91223961T>C", 91223961L, 91223961L, 1,
-                                                         "NT_455866.1:g.1766472T>C", 1766472L, 1766472L, 1));
-        matchingAllelesVariants.add(new SubSnpCoreFields(26963037L, 1, 13677177L, 1,
-                                                         "NT_455866.1", 1766472L, 1766472L, 1, LocusType.SNP,
-                                                         "4", 91223961L, 91223961L,
-                                                         "T", "T", "A", "T/A",
-                                                         "NC_006091.4:g.91223961T>A", 91223961L, 91223961L, 1,
-                                                         "NT_455866.1:g.1766472T>A", 1766472L, 1766472L, 1));
-        matchingAllelesVariants.add(new SubSnpCoreFields(0, -1, 0L, 1,
-                                                         "", 0L, 0L, 1, LocusType.SNP,
-                                                         "4", 0L, 0L,
-                                                         "T", "T", "C", "T/A/G",
-                                                         "", 0L, 0L, 1,
-                                                         "", 0L, 0L, 1));
-        matchingAllelesVariants.add(new SubSnpCoreFields(0, -1, 0L, 1,
-                                                         "", 0L, 0L, 1, LocusType.SNP,
-                                                         "4", 0L, 0L,
-                                                         "AT", "AT", "TGG", "TT/AT/CCA",
-                                                         "", 0L, 0L, 1,
-                                                         "", 0L, 0L, 1));
+        matchingAllelesVariants
```

Comment:
The formatting of the following blocks is pretty weird Is this caused by our code style template?

Reply:
I'm not sure, the splitting in lines in IntelliJ has different behaviors depending on how do you add/ remove parameters to a method. What I usually do when I see something with weird blocks is join all the lines and call the autoformat to split them properly. I'm updating those blocks ..

Reply:
Ok, I thought you were talking about splitting the arguments, but you were talking about mismatchinAllelesVariants in a single line. That was created by my code style template, that was originally copied from a Josemi one. I'm going to review it

Reply:
I've selected "Take priority over call chain wrapping" in the section "Method call arguments" anow lit looks better (see a7e57ce).